packages/botonic-core/src/index.js

```diff
@@ -1,21 +1,11 @@
 export { Providers } from './constants'
 export { CoreBot } from './core-bot'
 export * from './debug'
-export {
-  cancelHandoff,
-  deleteUser,
-  getAgentVacationRanges,
-  getAvailableAgents,
-  getAvailableAgentsByQueue,
-  getOpenQueues,
-  HandOffBuilder,
-  humanHandOff,
-  storeCaseRating,
-} from './handoff'
+export * from './handoff'
 export { HubtypeService } from './hubtype-service'
 export { getString } from './i18n'
 export { Router } from './router'
-export { isBrowser, isMobile, isNode, params2queryString } from './utils'
+export * from './utils'
```

Comment:
Why are those `*` imports necessary?

Reply:
It's a shorthand to export all the members exported in `./utils`, so instead to do `import {isBrowser, ...} from './utils'` and then export them, you can export all the functions directly with `*`

Reply:
they are exposed as `@botonic/react` uses them, also they can be useful for the developer.